Cloudflare waf modeling rules (#25699)
demisto/content: Packs/CloudflareWAF/README.md

- Retrieve all zones in account.
- Run a pipeline.

This pack contains an integration, whose main purpose is to manage firewall rules in Cloudflare Services.

In addition, this pack includes XSIAM content.

### Collect Events from Cloudflare WAF (XSIAM)

> We are currently support the retrieval of events from Cloudflare WAF by using an HTTP Log Collector, and Cloudflare waf logpush v2.

Reviewer: ```suggestion
We currently support the retrieval of events from Cloudflare WAF by using an HTTP Log Collector, and Cloudflare waf logpush v2.
```

**On XSIAM:**

> 1. Navigate to **Settings** -> **Data Sources** -> **Add Data Source** -> **Custom - HTTP based Collector** -> Click on **Connect Another Instance**, set the Name and Compression as you choose and then please set:

Reviewer: ```suggestion
1. Navigate to **Settings** -> **Data Sources** -> **Add Data Source** -> **Custom - HTTP based Collector**.
2. Click **Connect Another Instance**.
3. Set the Name and Compression and then set:
```

   - Name as `Cloudflare`
   - Compression `gzip`
   - Log Format as `JSON`
   - Vendor as `cloudflare`
   - Product as `waf`

> 2. Creating a new HTTP Log Collector will allow you to generate a unique token, please save it since it will be used later.

Reviewer: ```suggestion
2. Creating a new HTTP Log Collector allows you to generate a unique token. Save it since it will be used later.
```

> 3. Click the 3 dots sign next to the newly created instance and copy the API Url, it will also be used later.

Reviewer: ```suggestion
3. Click the 3 dots next to the newly created instance and copy the API URL. It will also be used later.
```

**On Cloudflare:**

> In order to configure the logpush on cloudflare side, please read [this documentation](https://developers.cloudflare.com/logs/get-started/enable-destinations/http/). 

Reviewer: ```suggestion
In order to configure the logpush on the Cloudflare side, read [this documentation](https://developers.cloudflare.com/logs/get-started/enable-destinations/http/). 
```

<u>Guidelines:</u>

> 1. For the **destination_conf**, use the API Url which was copied in section 3 (on the XSIAM side).

Reviewer: ```suggestion
1. For the **destination_conf**, use the API URL which was copied in section 3 (on the XSIAM side).
```

2. For the **X-Auth-Key**, use the newly created token mentioned in section 2 (on the XSIAM side).
3. **Important:**

>    Make sure to specify under **logpull_options** in the Logpush API configuration the next string `EdgeEndTimestamp&timestamps=rfc3339`.

Reviewer: ```suggestion
   Make sure to specify under **logpull_options** in the Logpush API configuration the string `EdgeEndTimestamp&timestamps=rfc3339`.
```

   This is in order to send this time field as a timestamp string, which will be used as the time of the event.

>    In addition, make sure to set the time zone of the logs to UTC (UTC+0).

Reviewer: ```suggestion
   In addition, make sure to set the timezone of the logs to UTC (UTC+0).
```
